app/assets/javascripts/controllers/container_dashboard/util.js

```diff
@@ -0,0 +1 @@
+angular.module('miq.util', []);
```

Comment:
Yes, it was just moved.. as for why..

(From the top-level PR description)
> miq.util needed to be moved because of loading order issues

(From comment addressed to you)
> I had to move the miq.util module definition so it would get loaded before the files that add stuff there.. I think defining the module in foo.js and defining its innards in foo/*.js makes some sense, but I something like keeping it in the same directory but making sure the name is alphabetically first would work too... or we could scratch the require_tree idea but.. I'd like to have that :)